src/main.js

```diff
@@ -511,7 +506,7 @@ function getAdditionalPaneInfo(contents) {
     }
     return {
       name: content["name"],
-      url: url,
+      url: url.href,
```

Comment:
```suggestion
      url: content["url"],
```

こっちのほうが周りとの統一感があって良くないです？

Reply:
あー、 tryのほうで`new URL` 試して正当性は確認済みだから、ここはもう元の `content["url"]` を渡すで問題なかったですね。ありがとうございます！